packages/components/src/components/link-pure/link-pure.tsx

```diff
@@ -53,7 +53,7 @@ export class LinkPure {
   @Element() public host!: HTMLElement;
 
   /** Aligns the label. */
-  @Prop() public alignLabel?: BreakpointCustomizable<LinkPureAlignLabel> = 'right';
+  @Prop() public alignLabel?: BreakpointCustomizable<LinkPureAlignLabel> = 'end';
```

Comment:
What about deprecation warning for user?

Reply:
As far as I know, we are not able to provide a deprecation warning for breakpoint customizable prop values.